Documentation/kubernetes/requirements.rst

```diff
@@ -0,0 +1,95 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _k8s_requirements:
+
+************
+Requirements
+************
+
+Kubernetes Version
+==================
+
+The following Kubernetes versions have been tested in the continuous integration
+system for this version of Cilium:
+
++---------------------+---------------------------------------------------------------+
+| Option              | Notes                                                         |
++---------------------+---------------------------------------------------------------+
+| 1.9                 |                                                               |
```

Comment:
Missing 1.8

Reply:
What is the point of this table? It seems to have no notes at all..?

Reply:
I figured I would have notes to add as I complete the other sections but that never happend ;-). I've converted the table into a list.